allennlp/models/basic_classifier.py

```diff
@@ -123,6 +125,25 @@ def forward(self,  # type: ignore
 
         return output_dict
 
+    @overrides
+    def decode(self, output_dict: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor]:
+        """
+        Does a simple argmax over the probabilities, converts index to string label, and
+        add ``"label"`` key to the dictionary with the result.
+        """
+        predictions = output_dict["probs"]
+        if predictions.dim() == 2:
+            predictions_list = [predictions[i] for i in range(predictions.shape[0])]
+        else:
+            predictions_list = [predictions]
+        classes = list()
```

Comment:
typically we'd just use `classes = []` here.

Reply:
Fixed it, will collate all changes requested and push new commit.